pocket_coffea/lib/weights_manager.py

```diff
@@ -139,6 +139,7 @@ def __init__(
         self._year = metadata["year"]
         self._finalstate = metadata["finalstate"]
         self._xsec = metadata["xsec"]
+        self._sum_genweights = float(metadata["sum_genweights"])
```

Comment:
For some reason, unlike `metadata["xsec"]` the parameter `metadata["sum_genweights"]` is a string so it needs to be casted into a float.